src/demo/share/java2d/J2DBench/src/j2dbench/report/J2DAnalyzer.java

```diff
@@ -270,7 +270,7 @@ public static void main(String[] argv) {
             } else {
                 double overallscore = totalscore[i]/numtests[i];
                 System.out.println("    Number of tests:  "+numtests[i]);
-                System.out.println("    Overall average:  "+overallscore);
+                System.out.printf("    Overall average:  %10.4f%n", overallscore);
```

Comment:
```
        double overallscore = 2.4322;
        System.out.println("    Overall average:  "+overallscore); // before
        System.out.printf("    Overall average:  %20.4f%n", overallscore); // after
```

leads to
```
    Overall average:  2.4322 // before
    Overall average:                2.4322 // after
```

should it be a fixed 10 width here?

Reply:
I am not sure what you mean, but with the 10, the scores line up with the rest of the minor stats when it is monospace font like this example -

```
Summary:
  vimg_text_noaa: 
    Number of tests:  2
    Overall average:  20002.7449
    Best spread:      1.3% variance
    Worst spread:     3.62% variance
    (Basis for results comparison)

```

Reply:
The scores are chars/sec or pixels/sec so it is always a big number with (usually) many decimal places. 
```
    Overall average:  19046.2340
    Overall average:  20010.8816
    Overall average:  19829.0079
    Overall average:  16595601.1584
    Overall average:  61195068.8361
```
I added a - to the format string to always left align the result which should work for big or small numbers.